admin/data/sql/migrations/20191112222703.undo.add-aliases-pupil-table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE mtc_admin.pupil DROP COLUMN lastNameAlias
```

Comment:
IMO, SQL should be terminated by a semi-colon.

Reply:
done